examples/complete/variables.tf

```diff
@@ -39,3 +39,9 @@ variable "existing_sm_instance_region" {
   description = "Required if value is passed into var.existing_sm_instance_guid"
   default     = null
 }
+
+variable "enable_cors" {
```

Comment:
Do not expose this in the example. We don't want example to become flexible solutions. I suggest to hard code it to true in the complete example main.tf with a code comment explaining what it does.